internal/common/satellite/satellite.go

```diff
@@ -31,9 +31,11 @@ func GetSatHostInfo(events []messageModel.PlaybookSatRunResponseMessageYamlEvent
 		if event.Host != nil && *event.Host != *host {
 			continue
 		}
-		if event.Sequence != nil {
+
+		if event.Sequence != nil && *event.Sequence >= hostInfo.Sequence {
```

Comment:
What if instead of doing this, we ordered the incoming events (based on the sequence field) before processing them?

Reply:
incoming events should now be ordered :+1: